src/std/fs.ab

```diff
@@ -76,3 +78,45 @@ pub fun change_owner(user: Text, path: Text): Bool {
 
     return false
 }
+
+/// Finds all files or directories matching a file glob.  Escapes all
+/// characters in the passed-in glob apart from "*", "?" and "/", to
+/// prevent injection attacks.
+pub fun glob(path: Text): [Text]? {
+    path = replace_regex(path, "\([^*?/]\)", "\\\\\1")
+    let files = $eval "for file in {path}; do [ -e \\\"\\\$file\\\" ] && echo \\\"\\\$file\\\"; done"$?
+    return split(files, "\n")
```

Comment:
Instead of duplicating this logic in both the regular and multiple versions of the function, lets extract it into a separate private function.

Reply:
Good call.